pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -96,7 +97,7 @@ func (s *initScaffolder) Scaffold() error {
 			DeployImages: len(imagesEnvVars) > 0,
 			HasWebhooks:  hasWebhooks,
 		},
-		&templatescertmanager.Certificate{},
+		&templatescertmanager.Certificate{HasWebhooks: hasWebhooks},
```

Comment:
sorry Camila, just wondered whether it is being updated due to scaffalding webhooks conditional issues.

Reply:
we are passing the value HasWebhooks here because then inside of the boilerplate to generate the Certs we just generate those which related to Webhook Sever only when we have Webhooks